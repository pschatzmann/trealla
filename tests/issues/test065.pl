:- initialization(main).

:- use_module(library(lists)).

main :-
	setof(I, member(I, [A,B,B,A]), Set), writeln(Set),
	bagof(I, member(I, [A,B,B,A]), Bag), writeln(Bag).
