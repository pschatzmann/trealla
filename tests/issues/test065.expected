[_2,_3]
[_2,_3,_3,_2]
